### PR TITLE
microsite: update config schema added switch db

### PR DESCRIPTION
**Pull request by ayocodingit**
## Overview

- microsite: update config schema added switch db

## Evidence
- title: microsite: update config schema added switch db
- project: Portal Jabar
- participants: @ayocodingit @muhamadabduh @rindibudiaramdhan @rachadiannovansyah @rhmnmbr

### CODE REVIEW

**Review by rindibudiaramdhan — APPROVED** [review]

**Review by rhmnmbr — APPROVED** [review]

File: microsite-service/src/database/mongo/schemas/page.ts [hunk 1]
```diff
@@ -38,4 +39,6 @@ schema.pre('save', function (next) {
     next()
 })
 
-export default mongoose.model('pages', schema)
+export default (database: string) => {
+    return Mongo.switch(database).model('pages', schema)
+}
```
Comment (muhamadabduh) [inline microsite-service/src/database/mongo/schemas/page.ts:44]: could you elaborate about the `switch` method of Mongo ? Is this part of the implementation of multi-db of microsite ?

**Review by rachadiannovansyah — APPROVED** [review]

File: microsite-service/src/database/mongo/schemas/page.ts (hunk 1 above)
Comment (ayocodingit) [reply microsite-service/src/database/mongo/schemas/page.ts:44]: Screenshot_20230207_083902_Chrome
For explanation a fully about architecture microsite

Comment (ayocodingit) [reply microsite-service/src/database/mongo/schemas/page.ts:44]: please review again Mr. @muhamadabduh

Comment (rindibudiaramdhan) [reply microsite-service/src/database/mongo/schemas/page.ts:44]: in database OPDs has similar master data? like `provinsi`, `kabkot` etc

Comment (ayocodingit) [reply microsite-service/src/database/mongo/schemas/page.ts:44]: not, for example, databases of `Diskominfo`, `Bkd` some like this

**Review by muhamadabduh — APPROVED** [review]

**Pull request merged**